saleor/graphql/account/mutations/authentication/request_password_reset.py

```diff
@@ -99,25 +106,41 @@ def clean_user(cls, email, redirect_url, info: ResolveInfo):
     def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
         email = data["email"]
         redirect_url = data["redirect_url"]
-        channel_slug = data.get("channel")
         user = cls.clean_user(email, redirect_url, info)
+        user.last_password_reset_request = timezone.now()
+        user.save(update_fields=["last_password_reset_request"])
```

Comment:
I'm not sure if we should save `last_password_reset_request` here; we run some validations in `post_save_action`, so wouldn't that lead to a situation where the new timestamp is saved, and later we get a validation error, which would effectively not send the notification?

Reply:
If there are validations which I missed, while changing the order of sending notifications with instance save, I will need to send notifications before save is done. However this will not be post_save_actions anymore (triggered before the actual save)

Reply:
@maarcingebala I have added changes, however I'm not sure if sending a notification and triggering webhooks should still be a part of post_save_actions - where they are triggered before save. Should I change it, moving them to separate function or just to save itself?

Reply:
Sending the event is the main point of this mutation, not a save. So I think only the naming "post_save" is confusing here. Maybe we don't need this function at all and everything should happen perform_mutation.

Reply:
We're overriding `perform_mutation` anyway, and `post_save_action` is mostly useful when you want to override just that "post save" part of the flow.

Reply:
My idea was to keep additional actions in one place, however maybe for this case it does not make sense to add extra method which naming is misleading just to keep all events calls in `post_save_action` as a pattern

Reply:
anyways, I added all the logic to the `perform_mutation` method cause it seems to be more convenient